src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/DefaultSaveWorkloadService.kt

```diff
@@ -47,8 +48,10 @@ class DefaultSaveWorkloadService(
 
     if (personManagerSaveResult.hasChanged || eventManagerSaveResult.hasChanged || requirementManagerSaveResults.any { it.hasChanged }) {
       notificationService.notifyAllocation(allocationData, allocateCase, caseDetails)
+      log.info("Allocation notified for case: ${caseDetails.crn}, conviction number: ${allocateCase.eventNumber}, to: ${allocationData.staff}, from: ${allocationData.allocatingStaff}")
```

Comment:
Just wondering, Do we want all the staff details here rather than just the name or code?